src/content_script.js

```diff
@@ -16,20 +16,33 @@ function findAllCustomElements(nodes) {
     if (isCustomElement(el) && el.localName !== 'style' && !allCustomElements.find(ce => ce === el.localName)) {
       allCustomElements.push(el.localName);
     }
-    
+
     if (el.shadowRoot) {
       findAllCustomElements(el.shadowRoot.querySelectorAll('*'));
     }
   }
 }
 
+chrome.runtime.onConnect.addListener(function(port) {
+  port.onMessage.addListener(function(msg) {
+    port.postMessage({counter: msg.counter+1});
+  });
+});
+
 chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
-  if(request.msg === "init") {
+
+  if(request.msg === "findAll") {
     allCustomElements = [];
     findAllCustomElements(document.querySelectorAll('*'));
     sendResponse(allCustomElements)
   }
 
+  if(request.msg === "init") {
+    // They are already finded.
```

Comment:
Can we change this to: `// Custom elements are already found`? 🙂

Reply:
Done!